Modules/CTP/src/CTPTrendingTask.cxx

```diff
@@ -74,63 +78,53 @@ void CTPTrendingTask::initCTP(Trigger& t)
     mCTPconfigFound = true;
   }
 
-  try {
-    mClassNames[0] = std::stof(mCustomParameters.at("minBias1Class", "default"));
-  } catch (const std::exception& e) {
+  mClassNames[0] = getFromExtendedConfig<string>(t.activity, mCustomParameters, "minBias1Class", "CMTVX-B-NOPF");
```

Comment:
You could write
```
mClassNames[0] = getFromExtendedConfig<string>(t.activity, mCustomParameters, "minBias1Class", mClassNamesDefault[0]);
```
to avoid a risk of mismatch in the default value.

Also valid for the other lines below.